ctapipe/tools/process.py

```diff
@@ -68,11 +68,15 @@ class ProcessorTool(Tool):
         default_value=False,
     ).tag(config=True)
 
+    stereo_combiner_configs = List(Dict()).tag(config=True)
```

Comment:
What are these for?
Are they not part of the reconstructor configs?

Reply:
Yes, leftover from a previous version. Not needed anymore.